src/puppetlabs/puppetdb/anonymizer.clj

```diff
@@ -31,11 +32,10 @@
    (contains? event "old_value")
    (contains? event "resource_type")))
 
-(defn report?
+(pls/defn-validated report? :- report/report-schema
```

Comment:
I think this is fine with me now, I see where AJ is coming from. Using report? is consistent at least with how we're treating the other data types. I don't think stringifying inline is notably cleaner.